cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -238,13 +251,18 @@ Future<Void> reconcile(ReconciliationState reconcileState)  {
             }
         }
 
+        // only when cluster is full KRaft we can avoid reconcile ZooKeeper and not having the automatic handling of
+        // inter broker protocol and log message format via the version change component
+        LOGGER.infoCr(reconcileState.reconciliation, "KafkaMetadataConfigurationState = {}, isKRaftEnabled = {}", kafkaMetadataConfigState, kafkaMetadataConfigState.isKRaft());
```

Comment:
Leftover log message from development?

Reply:
Yes, I will review all INFO level logging when we are close to accept the PR. They are still useful to me for development/testing during the review as well.